src/components/breadcrumbs/crumb/crumb.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import Crumb from "./crumb.component";
+
+const LINK_TEXT = "Link text";
+
+describe("Crumb", () => {
+  it("passes href to the anchor element isCurrent is false", () => {
+    render(<Crumb href="foo" text={LINK_TEXT} />);
+
+    const link = screen.getByText(LINK_TEXT);
+    expect(link).toBeInTheDocument();
+    expect(link?.closest("a")).toHaveAttribute("href", "foo");
```

Comment:
```suggestion
    expect(link.closest("a")).toHaveAttribute("href", "foo");
```
This assertion will fail if the link is not in the document anyway, so we can just use it on its own